### PR TITLE
shuttle docking autodoc

### PR DESCRIPTION
**Pull request by actioninja**
hey kids, remember autodoc?
No?
Oh.

### CODE REVIEW

**Comment by Fikou** [comment]
wow i can use that surgery thingy to implant a shuttle dock in myself?

**Comment by actioninja** [comment]
yes

**Comment by optimumtact** [comment]
they will remember it soon

**Comment by Ohagi-Chan** [comment]
What does this change do?

**Comment by optimumtact** [comment]
http://codedocs.tgstation13.org/ @Ohagi-Chan

File: code/modules/shuttle/shuttle.dm [hunk 1]
```diff
@@ -11,19 +11,28 @@
...
 	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
 	anchored = TRUE
-//
+	/// The identifier of the port or ship.
+	/// This will be used in numerous other places like the console,
+	/// stationary ports and whatnot to tell them your ship's mobile
+	/// port can be used in these places, or the docking port is compatible, etc.
 	var/id
-	// this should point -away- from the dockingport door, ie towards the ship
+	///this should point -away- from the dockingport door, ie towards the ship
```
Comment (ninjanomnom) [inline code/modules/shuttle/shuttle.dm]: Not necessarily true but is a common standard

File: code/modules/shuttle/shuttle.dm [hunk 2]
```diff
@@ -11,19 +11,28 @@
...
-	var/dwidth = 0	//position relative to covered area, perpendicular to dir
-	var/dheight = 0	//position relative to covered area, parallel to dir
+	///size of covered area, perpendicular to dir
+	var/width = 0
+	///size of covered area, parallel to dir
+	var/height = 0
+	///position relative to covered area, perpendicular to dir
+	var/dwidth = 0
+	///position relative to covered area, parallel to dir
+	var/dheight = 0
```
Comment (ninjanomnom) [inline code/modules/shuttle/shuttle.dm]: You don't need these modified in template mobile docks

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

File: code/modules/shuttle/shuttle.dm [hunk 3]
```diff
@@ -11,19 +11,28 @@
...
+	/// port can be used in these places, or the docking port is compatible, etc.
 	var/id
-	// this should point -away- from the dockingport door, ie towards the ship
+	///Common standard is for this to point -away- from the dockingport door, ie towards the ship
 	dir = NORTH
-	var/width = 0	//size of covered area, perpendicular to dir
-	var/height = 0	//size of covered area, parallel to dir
-	var/dwidth = 0	//position relative to covered area, perpendicular to dir
-	var/dheight = 0	//position relative to covered area, parallel to dir
+	///size of covered area, perpendicular to dir
```
Comment (ninjanomnom) [inline code/modules/shuttle/shuttle.dm]: This is also automatic for mobile

File: code/modules/shuttle/shuttle.dm [hunk 4]
```diff
@@ -11,19 +11,28 @@
...
+	var/width = 0
+	///size of covered area, parallel to dir
```
Comment (ninjanomnom) [inline code/modules/shuttle/shuttle.dm]: Ditto

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

**Review by ninjanomnom — APPROVED** [review]

**Pull request merged**